### DISCUSSION TITLE
Is it possible to have indicator in lightweight chart?

### DISCUSSION THREAD

**Discussion by big-javascript-fan** [tradingview/lightweight-charts#884] [General]
Hi everyone
I would like to have an indicator feature for my trading chart built with lightweight charting library.

Is there any way to implement it?
It is supported in the lightweight chart library?

Regards

**Comment by timocov** [comment]
Duplicate of #647

**Reply by big-javascript-fan** [reply]
Thank you so much. timocov.
May I get the sample code or any example?
Thanks
